docs/src/main/sphinx/release/release-375.md

```diff
@@ -0,0 +1,68 @@
+# Release 375 (24 Mar 2022)
+
+## General
+
+* Change behavior of `ALTER TABLE qualified_name TO unqualified_name` to leave
+  the table in the schema. This is backwards-incompatible behavioral change.
+  ({issue}`11282`)
+* Add processed input bytes and rows to query events in event listener. ({issue}`11623`)
+* Improve performance of queries that contain filter predicates on aggregation
+  results. ({issue}`11469`)
+* Fix `SHOW CREATE TABLE` to show actual table name in case of table
+  redirections, so that the statement can be used to recreate the table. ({issue}`11604`)
+* Fix scheduling for non-remotely accessible splits in fault-toleration execution. ({issue}`11581`)
+* Fix incorrect `is_nullable` value in `information_schema.columns` table. ({issue}`11613`)
+
+## JDBC driver
+
+* Allow `PreparedStatement.close()` to be called multiple times. ({issue}`11620`)
+* Fix incorrect `is_nullable` value in `DatabaseMetaData.getColumns()` method. ({issue}`11613`)
+
+## Delta Lake connector
+
+* Add access control to `drop_extended_stats` and `vacuum` procedures. ({issue}`11633`)
+* Fix incorrect query results when previous query coincides with drop table. ({issue}`11562`)
+
+## Hive connector
+
+* Fix infinite loop in the query optimizer when query contains predicates on a
+  struct field. ({issue}`11559`)
+
+## Iceberg connector
+
+* Fix query failure when query contains predicates on a struct field. ({issue}`11560`)
+* Fix query failure when reading from `$files` system table after a table column
+  has been dropped. ({issue}`11576`)
+
+## MongoDB connector
+
+* Support predicate pushdown on `boolean` columns. ({issue}`11536`)
+
+## MySQL connector
+
+* Add support for table comments. ({issue}`11211`)
+
+## Pinot connector
+
+* Fix handling of passthrough queries that include expressions containing
+  aggregation functions. ({issue}`9137`)
+* Fix incorrect results when aggregation functions on columns having
+  non-lowercase names are pushed down to Pinot. ({issue}`9137`, {issue}`10148`)
+* Fix possible incorrect results when grouping on columns of array types. ({issue}`9781`)
+
+## PostgreSQL connector
+
+* Improve performance of queries involving `OR` with `IS NULL`, `IS NOT NULL`
```

Comment:
I think we should make these four items into one item with a sub bullet list and separate issue links .. wdyt @findepi @martint ?

Reply:
I think it might become hard to understand. Would prefer keep separate.

Reply:
ok